src/boot/bootctl.c

```diff
@@ -1990,13 +1988,7 @@ static int install_random_seed(const char *esp) {
         if (!path)
                 return log_oom();
 
-        sz = random_pool_size();
```

Comment:
hmm, you are now hardcoding the size to 32bytes everywhere. which makes me wonder: for the other seed, the userspace one, should we stop bothering about random_pool_size() as well? 

what's the goal here? hardcode all pool sizes to 32? or use the size reported by the kernel everywhere (as before?). seems a bit strange to honour the kernel size for the userspace seed but not here?

Reply:
The EFI thing wants 32 bytes. And this is consistent with is hashing too.

The kernel's pool size also happens to be 32 bytes. We could move to hard coding this instead of reading it all the time, but that seems separate and I don't see a rush to do it.

Reply:
ok, makes sense.